js/menu.js

```diff
@@ -59,22 +59,28 @@ function init() {
       return;
     }
 
-    $('body').toggleClass('disableScroll circle-menu-active');
-    $(this).find('.hamburger').toggleClass('is-active');
-    $(this).toggleClass('active');
-    $('.fl-menu-circle-nav-holder .nav-circle').toggleClass('active');
-    $('.fl-menu-circle-nav-list-holder').toggleClass('active').scrollTop(0);
+    var $this = $(this);
```

Comment:
Give `this` or `$this` a name where possible.

```suggestion
    var $toggle = $(this);
```

Reply:
@tonytlwu renamed